lib/settings_page.dart

```diff
@@ -1,9 +1,15 @@
 import 'package:flutter/cupertino.dart';
```

Comment:
im confused on our naming convention, aren't we using camel case when naming our dart files?

Reply:
I actually forgot to change this, this was named prior to me joining, but I can definitely change it to camelCase.